Tests/Marketplace/search_and_uninstall_pack.py

```diff
@@ -0,0 +1,263 @@
+    logging.debug('Skipping packs uninstallation - nothing to uninstall')
+    return True
+
+
+def check_base_pack_version(client: demisto_client):
+    """
+
+        Args:
+            client (demisto_client): The client to connect to.
+
+        Returns:
+            list of installed python
+        """
+    try:
+        logging.info("Attempting to fetch installed Base pack.")
+        response_data, status_code, _ = demisto_client.generic_request_func(client,
+                                                                            path='/contentpacks/installed/Base',
+                                                                            method='GET',
+                                                                            accept='application/json',
+                                                                            _request_timeout=None)
+        if 200 <= status_code < 300:
+            installed_base = ast.literal_eval(response_data)
+            logging.success(f'Successfully fetched Base pack with version {installed_base.get("currentVersion")}')
+            return installed_base.get("currentVersion")
+        else:
+            result_object = ast.literal_eval(response_data)
+            message = result_object.get('message', '')
+            raise Exception(f'Failed to fetch Base pack - with status code {status_code}\n{message}')
+    except Exception as e:
+        logging.exception(f'The request to fetch installed packs has failed. Additional info: {str(e)}')
+        return None
+
+
+def reset_base_pack_version(client: demisto_client):
+    """
+    Resets base pack version to prod version.
+
+    Args:
+        client (demisto_client): The client to connect to.
+
+
+    """
+    host = client.api_client.configuration.host
+    try:
+        # make the search request
+        response_data, status_code, _ = demisto_client.generic_request_func(client,
+                                                                            path='/contentpacks/marketplace/Base',
+                                                                            method='GET',
+                                                                            accept='application/json',
+                                                                            _request_timeout=None)
+        if 200 <= status_code < 300:
+            result_object = ast.literal_eval(response_data)
+
+            if result_object and result_object.get('currentVersion'):
+                logging.debug('Found Base pack in bucket!')
+
+                current_installed_base_version = check_base_pack_version(client)
+                if current_installed_base_version is not None and current_installed_base_version == result_object.get(
+                        'currentVersion'):
+                    logging.debug('Installed Base pack is at the same version as the pack in the bucket.')
+                    return True
```

Comment:
@dantavori @eyalpalo 
I think we need to remove this. Consider the following scenario: 
I am changing `Base` pack. 
I am not bumping the pack version, because I am not done yet, just started to work on this pr.. (And because it is very annoying to generate-release-notes to Base pack, since every merge from master it will be a conflict, that someone already bumped the version of this pack)... 
I am pushing my changes to github, and the build starting to run, new bucket created with updated Base pack, we installing Base pack from new-custom bucket, the code is different but the version is same as in the production!!!!!!
The build done. 
Next developer opens a pr, runs build, this script runs and it checks if the Base pack version is the same as in the production, yes its the same, ok then do not uninstall it. 
It creates a problem where next build runs with Base pack not same as in the production.

```suggestion
```